CHANGELOG.rst

```diff
@@ -1,8 +1,12 @@
 Changelog
 =========
 
-3.2.0
-------------------
+3.2.1
+--------------------
+    - Fix rST formatting in this file to pass PyPI rendering check
+
+3.2.0 (Not released)
+--------------------
     - Support Django 2.3
```

Comment:
It's about Django **2.2** support, not 2.3.

`DJANGO_VERSION_CEILING` is used as:
>  Django **<** `DJANGO_VERSION_CEILING`

I was confused by meaning of `DJANGO_VERSION_CEILING`, too :-)

Reply:
Thanks for the catch!